index.bs

```diff
@@ -3378,8 +3378,9 @@ The [=set/items=] of an [=effective overload set=] are [=tuples=] of the form
 whose [=tuple/items=] are described below:
 
 *   A <dfn for="effective overload set tuple">callable</dfn> is an [=operation=]
-    if the [=effective overload set=] is for [=regular operations=] or [=static operations=]; and
-    it is an [=extended attribute=] if the [=effective overload set=] is for constructors or [=named constructors=].
+    if the [=effective overload set=] is for [=regular operations=] or
+    [=static operations=]; and it is either an [=operation=] or an [=extended attribute=]
+    if the [=effective overload set=] is for [=constructor operations=] and [=named constructors=].
```

Comment:
The extended attribute bit now no longer applies to constructors, only to named constructors. Constructor operations should move to the first clause.

Reply:
Ah, cool. Yeah, I was fuzzy on whether Named + op could be treated as members of a single overload set.